src/plugins/TickingClockPlugin.cpp

```diff
@@ -0,0 +1,64 @@
+#include "plugins/TickingClockPlugin.h"
+
+void TickingClockPlugin::setup()
+{
+  // loading screen
+  Screen.setPixel(4, 7, 1);
+  Screen.setPixel(5, 7, 1);
+  Screen.setPixel(7, 7, 1);
+  Screen.setPixel(8, 7, 1);
+  Screen.setPixel(10, 7, 1);
+  Screen.setPixel(11, 7, 1);
+
+  previousMinutes = -1;
+  previousHour = -1;
+}
+
+void TickingClockPlugin::loop()
+{
+  if (getLocalTime(&timeinfo))
+  { 
+
+    if ( (timeinfo.tm_hour*60+timeinfo.tm_min) < 6*60+30  || 
+         (timeinfo.tm_hour*60+timeinfo.tm_min) > 22*60+30 ) //only between 6:30 and 22:30
```

Comment:
Thanks for the rewiew, no doubts, constants would be better then what i did here, however:

- Actually i would as well like to see this as a global feature, not something plugin specific. I would happily add it globaly, if @ph1p agrees, but i think this would be a different PR
- And i believe it should not be constant but needs to be configurable, so once the PR with the wifi manager is mereged, i could add an additional user parameter there. 

So, for now it is just a local quick fix in that plugin to keep my significant other happy. :-)

Best
Michael